src/pydvl/valuation/methods/owen_shapley.py

```diff
@@ -1,7 +1,73 @@
+from __future__ import annotations
+
+from pydvl.utils import Status
+from pydvl.valuation.methods.semivalue import SemivalueValuation
+from pydvl.valuation.samplers.powerset import OwenSampler
+from pydvl.valuation.stopping import NoStopping
+from pydvl.valuation.utility.base import UtilityBase
+
 """
 ## References
 
 [^1]: <a name="okhrati_multilinear_2021"></a>Okhrati, R., Lipani, A., 2021.
     [A Multilinear Sampling Algorithm to Estimate Shapley Values](https://ieeexplore.ieee.org/abstract/document/9412511).
     In: 2020 25th International Conference on Pattern Recognition (ICPR), pp. 7992–7999. IEEE.
 """
+
+
+class OwenShapleyValuation(SemivalueValuation):
+    """Umbrella class to calculate least-core values with Owen sampling schemes.
```

Comment:
You could add a link to the documentation here, describing what Owen Shapley values are. But I guess the documentation is outdated by now, correct? Do you want to update the documentation in this PR?

Reply:
We'll do the update as part of the re-structuring.